azure_costs_exporter/prometheus_collector.py

```diff
@@ -68,6 +68,10 @@ def _get_azure_data(self, month=None):
         rsp = requests.get(url, headers=headers, timeout=10)
         rsp.raise_for_status()
 
+        if rsp.text.startswith('"Usage Data Extract"'):
```

Comment:
With the original version we'll depend on the implementation detail of Azure for empty months. But I bet (hope) that this will be quite stable. The benefit to escalating other generic errors in the output outweighs that drawback IMO. Let's see if I am proven wrong ;-)